### ISSUE TITLE
Move e2e to unit test framework

### ISSUE THREAD

**Issue by JonasKellerer** [GenSpectrum/LAPIS-SILO#317]
In #290 we introduced a new testing framework for queries to test the queries in unit tests. Use this framework to replace the e2e tests in `endToEndTests/test/queries`.

- [] #

**Comment by fengelniederhammer** [comment]
We should not move all tests. I think the criterium should be: 
There should be at least one example for every kind of query that demonstrates how to send the query and what the result looks like.

Testing edge cases and special logic should however be moved to unit tests.

**Comment by Taepper** [comment]
I do not see a downside to keeping the e2e tests. Any migration should only be performed if the unit tests are viable enough to read the json definition of the e2e testcases, which are more descriptive than the in-code format of the current unit test format

**Comment by Taepper** [comment]
That is true. I like a non-code format for test cases, where the files are as descriptive as possible. Not needing to repeat all data in each testcase is another point one should consider. I think we are on a good track to achieve that, where only a few more bolts and screws need to be adjusted
